test/project_types/extension/extension_test_helpers/dummy_argo_setup.rb

```diff
@@ -0,0 +1,13 @@
+module Extension
+  module ExtensionTestHelpers
+    class DummyArgoSetup < Extension::Features::Argo::Base
```

Comment:
I think `Setup` should be removed from the class name to be consistent with the naming of other subclasses e.g. `Checkout`, `Admin` that inherit from `Base`.

Reply:
I noticed you updated the name in the `BaseTest`, but I think this class should also be renamed `DummyArgo` to fall in line with other types.